axelrod/classifier.py

```diff
@@ -0,0 +1,205 @@
+from typing import Any, Callable, Generic, List, Optional, Set, Text, Type, \
+    TypeVar, Union
+
+import os
+import yaml
+
+from axelrod.player import Player
+
+ALL_CLASSIFIERS_PATH = "data/all_classifiers.yml"
+
+T = TypeVar('T')
+
+
+class Classifier(Generic[T]):
```

Comment:
I see what you're saying, and I agree that it's usually better to avoid creating a class if we can help it.  However, in this case, I slightly lean towards how I have it, for a few reasons:
1. I like the way we declare type here.
2. We can work with keys and values in a dict, but I kinda like having these as variables named as "player_class_classifier" and "name".
3. It's imaginable that this would be expanded at some point.

Reply:
ok